### PR TITLE
add FreeBSD and SLES specific notes

### LINKED ISSUE #315: SLES provisioning
Perhaps part of section 4.4 (provisioning), but some detail around SLES provisioning (#3733) would be good.

### LINKED ISSUE #316: FreeBSD provisioning
Perhaps part of section 4.4 (provisioning), but some detail around FreeBSD provisioning and its requirements (#2572) would be good.

### PR DESCRIPTION
**Pull request by mmoll**
closes GH-315
closes GH-316

### CODE REVIEW

File: _includes/manuals/1.9/4.4.8.2_prov_sles.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+The installation media URL has to contain the contents of the first SLES DVD, it's easiest to loopback mount the ISO image on a webserver. For Puppet, the `systemsmangement:puppet` repository on OBS are used.
+
+##### SLES 11
+For Puppet, in addition to `systemsmangement:puppet` also the `devel:languages:ruby:backports` repository on OBS and the SLES SDK DVD is used. The placeholder in the AutoYaST SLES template has to be updated with the actual SDK URL.
```
Comment (dLobatog) [inline _includes/manuals/1.9/4.4.8.2_prov_sles.md]: Typo, `systemsmanagement:puppet` , missing an `a` in between `n` and `g`. Same on line 1

**Comment by dLobatog** [comment]
I guess it's worth emailing the [FreeBSD foundation](https://www.freebsdfoundation.org/documents/Guidelines.shtml) We can probably use their logo with permission, but they say variations on their mark (like the F on black background) have to be approved.

File: _includes/manuals/1.9/4.4.8.2_prov_sles.md (hunk 1 above)
Comment (mmoll) [inline _includes/manuals/1.9/4.4.8.2_prov_sles.md]: fixed.

**Comment by mmoll** [comment]
The F icon was taken from https://github.com/lzap/foreman-dummy-icons and is not a variation of the FreeBSD logo. I'd like to get this in as-is and when it's online I can write a mail from my FreeBSD.org address to ask them if it's OK if we use some grey variation of the original logo.

File: _includes/manuals/1.9/4.4.8_prov_os_notes.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This chapter contains informations about provisioning specific operating systems.
```
Comment (domcleal) [inline _includes/manuals/1.9/4.4.8_prov_os_notes.md]: "information", singular

File: _includes/manuals/1.9/4.4.8.2_prov_sles.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+The installation media URL has to contain the contents of the first SLES DVD, it's easiest to loopback mount the ISO image on a webserver. For Puppet, the `systemsmanagement:puppet` repository on OBS are used.
```
Comment (domcleal) [inline _includes/manuals/1.9/4.4.8.2_prov_sles.md]: "repository on OBS is used"

File: _includes/manuals/1.9/4.4.8_prov_os_notes.md (hunk 2 above)
Comment (mmoll) [inline _includes/manuals/1.9/4.4.8_prov_os_notes.md]: done

File: _includes/manuals/1.9/4.4.8.2_prov_sles.md (hunk 3 above)
Comment (mmoll) [inline _includes/manuals/1.9/4.4.8.2_prov_sles.md]: done

**Comment by domcleal** [comment]
Thanks @mmoll!

**Pull request merged**